lib/runBlock.js

```diff
@@ -177,7 +177,7 @@ module.exports = function (opts, cb) {
           error: err
         }
 
-        afterBlock(cb.bind(this, new Error(err), result))
+        afterBlock(cb.bind(this, err, result))
```

Comment:
Is the input `err` to this method (`parseBlockResults`) always an `Error` instance?

Reply:
I think this was changed in the last weeks on many places to always use an ``Error`` object and not a plain string, wasn't it? Not sure though if this was a concerted effort to change this on all places.

It's probably nevertheless not a good solution to start with checks like

```
if (typeof(err) === 'string') {
  err = new Error(err)
}
```

at this place.

Reply:
The `Error`/string types seem compatible in node and the browser, so I guess this is fine.

Reply:
@holgerd77 I meant that this function receives `err` and only sets it to an `Error` in certain places, but there are no safeguards in place that the incoming `err` is an instance of `Error`.